src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/settings/ManagedIndexSettings.kt

```diff
@@ -24,9 +24,10 @@ import java.util.function.Function
 class ManagedIndexSettings {
     companion object {
         const val DEFAULT_ISM_ENABLED = true
-        const val DEFAULT_JOB_INTERVAL = 5
+        const val DEFAULT_JOB_INTERVAL = 1
```

Comment:
Is this intentional or just leftover from testing?

Reply:
it's just leftover

Reply:
chang back